staff/index.md

```diff
@@ -6,45 +6,39 @@ group: "navigation"
 
 # Staff Directory
 
-Name | Position | E-mail | Phone Number
---- | --- | --- | --- | ---
-[Taylor, Karl][taylor] | Program Director |	taylor13@llnl.gov |	(925) 423-3623
-[Ames, Sasha][ames] | Computer Scientist | ames4@llnl.gov	| (925) 424-4644
-[Bonfils, Celine][bonfils] | Research Scientist | bonfils2@llnl.gov | (925) 423-9923
-[Caldwell, Peter][caldwell] | Research Scientist | caldwell19@llnl.gov | (925) 422-4197
-[Covey, Curt][covey] | Research Scientist | covey1@llnl.gov	| (925) 422-1828
-[Doutriaux, Charles][doutriaux] | Computer Scientist	| doutriaux1@llnl.gov	| (925) 422-1487
-[Durack, Paul][durack] | Research Scientist	| durack1@llnl.gov	| (925) 422-5208
-[Gleckler, Peter][gleckler] | Research Scientist | gleckler1@llnl.gov	| (925) 422-7631
-[Harris, Matthew][harris] | Computer Scientist | harris112@llnl.gov	| (925) 423-8978
-[Hoang, Anthony][hoang] | Systems Manager	| hoang1@llnl.gov	| (925) 423-6733
-[Jefferson, Angela][jefferson] | Program Administrator | jefferson9@llnl.gov | (925) 422-5932
-[Lee, Jiwoo][lee] | Research Scientist | lee1043@llnl.gov | (925) 422-0932
-[Nadeau, Denis][nadeau] | Computer Scientist | nadeau1@llnl.gov | (925) 423-3123
-[Painter, Jeff][painter] | Computer Scientist	| painter1@llnl.gov	| (925) 422-8114
-[Phillips, Tom][phillips] | Research Scientist | phillips14@llnl.gov | (925) 422-0072
-[Santer, Ben][santer] | Research Scientist | santer1@llnl.gov	|
-[Sperber, Ken][sperber] | Research Scientist | sperber1@llnl.gov | (925) 422-7720
-[Williams, Dean][williams] | Computer Scientist	| williams13@llnl.gov	| (925) 423-0145
-[Xie, Shaocheng][xie] | Research Scientist	| xie2@llnl.gov	| (925) 422-6023
+Name | Position 
+--- | --- 
+[Taylor, Karl][taylor] | Program Director 
+[Ahn, Min-Seop][ahn] | Post-doc 
+[Ames, Sasha][ames] | Computer Scientist 
+[Bonfils, Celine][bonfils] | Research Scientist 
+[Caldwell, Peter][caldwell] | Research Scientist 
+[Covey, Curt][covey] | Research Scientist 
+[Durack, Paul][durack] | Research Scientist	
+[Gleckler, Peter][gleckler] | Research Scientist 
+[Hoang, Anthony][hoang] | Systems Manager	
+[Jefferson, Angela][jefferson] | Program Administrator 
+[Lee, Jiwoo][lee] | Research Scientist 
+[Painter, Jeff][painter] | Computer Scientist	
+[Phillips, Tom][phillips] | Research Scientist 
+[Santer, Ben][santer] | Research Scientist 
+[Sperber, Ken][sperber] | Research Scientist 
+[Xie, Shaocheng][xie] | Research Scientist
```

Comment:
It seems you removed the directory that this points to, Shaocheng is not part of the PCMDI program